spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -1269,9 +1378,10 @@ def config_options(self):
         cfg._merge(spy_cfg)
         return cfg
 
-    def interpreter_versions(self):
+    def interpreter_versions(self, is_environment=False, env=''):
```

Comment:
I think we could use here also just one `kwarg` -> `path_to_custom_interpreter=None`